test/bootstrap.php

```diff
@@ -11,6 +11,8 @@
     $include_path = $basedir . '/vendor' . PATH_SEPARATOR . ini_get('include_path');
     ini_set('include_path', $include_path);
 
+    putenv("ICINGAWEB_MODULES_DIR=$basedir/test/config/modules");
```

Comment:
that's not a module path, it's config

Reply:
Ahh, but without this fix, I got the following error:
```php
Module path "/usr/share/icingaweb2-modules/businessprocess/test/modules" does not exist
```

Reply:
However, the GitHub actions don't seem to care about that.